### ISSUE TITLE
implement a 'quick connect' feature

### ISSUE THREAD

**Issue by scarf** [znc/znc#977]
I would like to have the ability to quickly connect to any server, without having to configure anything (i.e. adding a new network).  For example, "/znc QuickConnect irc.server.net 6667" or something, which would just use all of the default network settings, and would be intended for single-use connections to IRC networks from time to time.  Optionally, for example, a "/znc SaveNetwork" command could also be implemented if the user then decides they wish to save this network connection in the config for future use.

**Comment by Mikaela** [comment]
But the IRC client would still have to connect to ZNC again to get to that server, so is it so useful?

**Comment by scarf** [comment]
yes it's much quicker.... I just type one command and i'm connected, instead of: creating a network, connect to the znc, adding a server, then connect to the server; or having to go through the web interface.  And then if i'm just using it once that clutters up the znc configuration

**Comment by Mikaela** [comment]
So by using that hypothetical command you disconnect from the network that you are already on?

**Comment by scarf** [comment]
yes, naturally.  or open a new tab/window/whatever if you want to have multiple connections

**Comment by Mikaela** [comment]
In that case it doesn't seem like very useful to me.

**Comment by dgw** [comment]
It's just as useful as using /znc JumpNetwork to change to another network
on the same connection.

**Comment by Mikaela** [comment]
Which I think is useless and the only time anyone has talked about that is support question at IRC on how does one connect to multiple networks at the same time and `/znc jumpnetwork` disconnects me from the first one.

**Comment by dgw** [comment]
Well, I use it on a regular basis to switch to networks that I don't use enough to have configured on e.g. mobile clients. It's plenty useful.

**Comment by scarf** [comment]
no because that all still requires a network to be configured first in ZNC and then at least one server added, nickname defined, etc.  i imagine one command to just connect to any server i want without having to do anything else.  i could do this with PsyBNC and Ezbounce and found it useful
